templates/web/fixmystreet.com/footer_extra.html

```diff
@@ -0,0 +1,87 @@
+                        <li><[% IF c.req.uri.path == '/reports' %]span[% ELSE %]a href="/reports"[% END
+                            %]>[% loc("All reports") %]</[% c.req.uri.path == '/reports' ? 'span' : 'a' %]></li>
+                        <li><[% IF c.req.uri.path == '/alert' %]span[% ELSE %]a href="/alert[% pc ? '/list?pc=' : '' %][% pc | uri %]"[% END
+                            %]>[% loc("Local alerts") %]</[% c.req.uri.path == '/alert' ? 'span' : 'a' %]></li>
+                    </ul>
+                    <ul>
+                        <li><a href="https://www.fixmystreet.com/council">FixMyStreet for Councils</a></li>
+                        <li><[% IF c.req.uri.path == '/posters' %]span[% ELSE %]a href="/posters"[% END
+                            %]>[% loc("FixMyStreet Goodies") %]</[% c.req.uri.path == '/posters' ? 'span' : 'a' %]></li>
+                        <li><[% IF c.req.uri.path == '/contact' %]span[% ELSE %]a href="/contact"[% END
+                            %]>[% loc("Contact") %]</[% c.req.uri.path == '/contact' ? 'span' : 'a' %]></li>
+                        <li><[% IF c.req.uri.path == '/faq' %]span[% ELSE %]a href="/faq"[% END
+                            %]>[% loc("Help") %]</[% c.req.uri.path == '/faq' ? 'span' : 'a' %]></li>
+                        <li><[% IF c.req.uri.path == '/privacy' %]span[% ELSE %]a href="/privacy"[% END
+                            %]>[% loc("Privacy") %]</[% c.req.uri.path == '/privacy' ? 'span' : 'a' %]></li>
+                    </ul>
+                </nav>
+            </div>
+
+            <div class="col-sm-3">
+                <div class="mysoc-footer__donate">
+                    <p>Your donations keep this site and others like it running</p>
+                    <a href="https://www.mysociety.org/donate" class="mysoc-footer__donate__button">Donate now</a>
+                </div>
+            </div>
+
+        </div>
+        <hr class="mysoc-footer__divider" role="presentation">
+        <div class="row">
+
+            <div class="col-sm-5">
+                <div class="mysoc-footer__orgs">
+                    <p class="mysoc-footer__org">
+                        Built by
+                        <a href="https://www.mysociety.org" class="mysoc-footer__org__logo mysoc-footer__org__logo--mysociety">mySociety</a>
+                    </p>
+                    <p class="mysoc-footer__org">
+                        Powered by
+                        <a href="https://fixmystreet.org" class="mysoc-footer__org__logo mysoc-footer__org__logo--fms-platform">FixMyStreet Platform</a>
+                    </p>
+                </div>
+            </div>
+
+            <div class="col-sm-4">
+                <div class="mysoc-footer__legal">
+                    <p>mySociety Limited (05798215) is a project of UK Citizens Online Democracy (UKCOD) a registered charity in England and Wales, charity number 1076346, company number 03277032.</p>
```

Comment:
Actually, no drop it. If this is needed, it's not enough - you need geographic address according to http://www.out-law.com/page-431 - and according to http://www.theregister.co.uk/2006/12/21/new_web_email_regulation/ about us type pages are sufficient.